admin/differ/DifferModel.js

```diff
@@ -77,12 +90,12 @@ export class DifferModel  {
                     actions: [this.applyRemoteAction]
                 },
                 {field: 'status', hidden: true},
-                {field: 'name', width: 200},
-                {
-                    field: 'type',
-                    width: 80,
-                    renderer: this.valueTypeRenderer
-                },
+                ...this.columnFields.map(field => {
+                    const ret = {field, renderer: this.fieldRenderer};
+                    if (field === 'valueType') ret.headerName = 'Type';
```

Comment:
I know this differ stuff is only used internally right now so this is probably fine, but having these checks for specific fields which come from specific types of records seems wrong...

Reply:
Maybe we allow providing column defs instead of just fields? Can keep the support for building the def from a field if they pass a string to make the api easy to use

Reply:
Makes sense - will push an update shortly